tools/pythonpkg/src/include/duckdb_python/numpy/numpy_type.hpp

```diff
@@ -12,34 +12,43 @@
 #include "duckdb_python/pybind11/pybind_wrapper.hpp"
 
 namespace duckdb {
+
 // Pandas has two different sets of types
 // NumPy dtypes (e.g., bool, int8,...)
 // Pandas Specific Types (e.g., categorical, datetime_tz,...)
 enum class NumpyNullableType : uint8_t {
 	//! NumPy dtypes
-	BOOL,      //! bool_, bool8
-	INT_8,     //! byte, int8
-	UINT_8,    //! ubyte, uint8
-	INT_16,    //! int16, short
-	UINT_16,   //! uint16, ushort
-	INT_32,    //! int32, intc
-	UINT_32,   //! uint32, uintc,
-	INT_64,    //! int64, int0, int_, intp, matrix
-	UINT_64,   //! uint64, uint, uint0, uintp
-	FLOAT_16,  //! float16, half
-	FLOAT_32,  //! float32, single
-	FLOAT_64,  //! float64, float_, double
-	OBJECT,    //! object
-	UNICODE,   //! <U1, unicode_, str_, str0
-	DATETIME,  //! datetime64[D], datetime64
-	TIMEDELTA, //! timedelta64[D], timedelta64
+	BOOL,        //! bool_, bool8
+	INT_8,       //! byte, int8
+	UINT_8,      //! ubyte, uint8
+	INT_16,      //! int16, short
+	UINT_16,     //! uint16, ushort
+	INT_32,      //! int32, intc
+	UINT_32,     //! uint32, uintc,
+	INT_64,      //! int64, int0, int_, intp, matrix
+	UINT_64,     //! uint64, uint, uint0, uintp
+	FLOAT_16,    //! float16, half
+	FLOAT_32,    //! float32, single
+	FLOAT_64,    //! float64, float_, double
+	OBJECT,      //! object
+	UNICODE,     //! <U1, unicode_, str_, str0
+	DATETIME_S,  //! datetime64[s], <M8[s]
+	DATETIME_MS, //! datetime64[ms], <M8[ms]
+	DATETIME_NS, //! datetime64[ns], <M8[ns]
+	DATETIME_US, //! datetime64[us], <M8[us]
+	TIMEDELTA,   //! timedelta64[D], timedelta64
 
 	//! ------------------------------------------------------------
 	//! Extension Types
 	//! ------------------------------------------------------------
-	CATEGORY,    //! category
-	DATETIME_TZ, //! datetime64[ns, TZ]
```

Comment:
Why was this removed?

Reply:
It's removed in favor of `has_timezone`, previously we only ever received a single time unit from pandas, ns, now that we can receive multiple we have to remember which unit this is, together with the fact that it has timezone information